### ISSUE TITLE
startGeofences() missing from TypeScript definitions

### ISSUE THREAD

**Issue by nickcoury** [transistorsoft/nativescript-background-geolocation-lt#80]
## Your Environment
* Plugin version: 1.6.0
* Platform: iOS
* OS version: 10.3.2
* Device manufacturer / model: iPhone SE
* Nativescript version (`tns info`): 3.0
* Plugin config

## Expected Behavior
`BackgroundGeolocation.startGeofences()` should be available.

## Actual Behavior
`BackgroundGeolocation.startGeofences()` is not in the TypeScript definitions, and it doesn't appear in `background-geolocation.ios.ts` where I expected it to be with the rest of the geofencing methods.  Instead, `startBackgroundTask` is in the slot it should appear in.

## Steps to Reproduce
```
        if (!this._state.enabled) {
            BackgroundGeolocation.startGeofences(state => {
                this._state = state;
                console.log('- Geofence-only monitoring started', state.trackingMode);
            });
        }
```
Code assistance in Visual Studio Code throws a TypeScript error because the method doesn't exist, and type definitions don't have `startGeofences` anywhere.  Searching through the repo, I don't find expected references to it in the Javascript code either (but do see it in the iOS code).

When ignoring the TypeScript error and running anyways, it indeed appears the method isn't present:
```
CONSOLE ERROR file:///app/tns_modules/@angular/core/./bundles/core.umd.js:1091:24: ERROR Error: Uncaught (in promise): TypeError: nativescript_background_g
eolocation_lt_1.BackgroundGeolocation.startGeofences is not a function. (In 'nativescript_background_geolocation_lt_1.BackgroundGeolocation.startGeofences'
, 'nativescript_background_geolocation_lt_1.BackgroundGeolocation.startGeofences' is undefined)
```

## Context
I believe this is an error and the method needs to be added to `background-geolocation.ios.ts`, but can't confirm as maybe I'm missing something.

**Comment by christocracy** [comment]
This method def could be missing.  I'll look into it next week (holiday weekend here).

**Comment by nickcoury** [comment]
Thanks.

In the mean time, I worked around it by calling into the library directly if anyone needs geofences before its exposed publicly:

```
BackgroundGeolocation.getLocationManager().startGeofences(); 
```

**Comment by christocracy** [comment]
Fixed in `1.7.0`.  See #87
